doc/user/using-java9.md

```diff
@@ -0,0 +1,34 @@
+# Using TruffleRuby with JDK 9
+
+TruffleRuby is designed to be run with a JVM that has the Graal compiler. The
+easiest way to do this is to use the GraalVM, which includes a JDK, the Graal
+compiler, and TruffleRuby, all in one package.
+
+It is also possible to run on Java 9, but Graal needs to be built with Java 9
+and there is no binary distribution of GraalVM based on Java 9 so you will
+need to [build Graal yourself](../contributor/building-graal.md), using Java
+9 as the JDK.
+
+```
+$ GRAAL_HOME=...path to graal repository built with JDK 9...
```

Comment:
GRAAL_HOME in all other contexts is the path to `.../graal/compiler` but here it seems to assume `.../graal`.
Could we use the conventional `.../graal/compiler` here too?